packages/require/lib/types.ts

```diff
@@ -14,7 +14,7 @@ export interface RequireOptions {
   /**
    * The `@truffle/config` for the user's project
    */
-  config: TruffleConfig;
+  config?: TruffleConfig;
```

Comment:
Can you explain why this is optional?

Reply:
Because that is how we use it https://github.com/trufflesuite/truffle/blob/develop/packages/migrate/Migration.js#L35-L42

Reply:
thanks!